### ISSUE TITLE
[PM-1349] Improve error message when jobs fail due to deep dir. structure with depth of > 20.

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1463]
pegasus-analyzer's output

```
Job 139.0 going into Hold state (code 12,1): Error from slot1@localhost: STARTER at 127.0.0.1 failed to send file(s) to <127.0.0.1:9618>; remaps resulted in a cycle: <0: _condor_stderr><1: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process/run0001//00/00/ls_ID0000001.err><2: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process/run0001//00/00><3: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process/run0001//00><4: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process/run0001/><5: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process/run0001><6: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus/process><7: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund/pegasus><8: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit/mukund><9: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc/submit><10: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test/abc><11: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples/test><12: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples/simple-examples><13: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus/examples><14: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share/pegasus><15: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev/share><16: /home/mukund/temp/pegasus_mybranch/dist/pegasus-5.0.0dev><17: /home/mukund/temp/pegasus_mybranch/dist><18: /home/mukund/temp/pegasus_mybranch><19: /home/mukund/temp><20: /home/mukund><abort>
```

**Reporter:** @mayani
**Resolution:** Won't Fix
**Watchers:**
@rynge
@mayani
@mukundmurrali
@vahi
**Attachments:**

**Comment by mayani** [comment]
**Author:** @mukundmurrali

@rajiv: Please find the tar in the attachment.

**Comment by mayani** [comment]
**Author:** @vahi

has to be handled at condor level
they have fixed it for 8.8.9

https://htcondor-wiki.cs.wisc.edu/index.cgi/tktview?tn=7581
